bigchaindb/common/schema/transaction.yaml

```diff
@@ -0,0 +1,245 @@
+---
+type: object
+additionalProperties: false
+title: Transaction Schema
+description: |
+    This is the outer transaction wrapper. It contains the ID, version and the body of the transaction, which is also called `transaction`.
+properties:
+  id:
+    pattern: "[0-9a-f]{64}"
```

Comment:
I think this can be replaced by the `sha3_hexdigest` definition below, along with moving the description to down there?